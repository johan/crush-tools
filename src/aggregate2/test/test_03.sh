test_number=03
description="multi-key, float sum"

infile=$test_dir/test.in
outfile=$test_dir/test_$test_number.out
expected=$test_dir/test_$test_number.expected

$bin -p -k 1,2 -s 5 $infile > $outfile

if [ $? -ne 0 ] ||
   [ "`diff -q $outfile $expected`" ]; then
  test_status $test_number 1 "$description" FAIL
else
  test_status $test_number 1 "$description" PASS
  rm "$outfile"
fi
