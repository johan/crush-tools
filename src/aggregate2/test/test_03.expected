Text-1	Text-2	Numeric-3
first text value	a	7.14
first text value	b	10.00
second text value	b	13.333
second text value	c	14.900
second text value	d	1.000
